cmd/kubeadm/app/cmd/reset.go

```diff
@@ -62,8 +62,10 @@ func NewCmdReset(in io.Reader, out io.Writer) *cobra.Command {
 			kubeadmutil.CheckErr(err)
 
 			kubeConfigFile = cmdutil.FindExistingKubeConfig(kubeConfigFile)
-			client, err = getClientset(kubeConfigFile, false)
-			kubeadmutil.CheckErr(err)
+			if _, err := os.Stat(kubeConfigFile); !os.IsNotExist(err) {
```

Comment:
Should we test just `err != nil`?

Reply:
I don't think it's safe to continue with reset if we encounter something different than ErrNotExist. The rest of the [stat(2)](http://man7.org/linux/man-pages/man2/stat.2.html) possible errors are bad enough for us to stop after the call to `getClientset`.

Reply:
@rosti 
> The rest of the [stat(2)](http://man7.org/linux/man-pages/man2/stat.2.html) possible errors are bad enough for us to stop after the call to `getClientset`

isn't is what kubeadmutil.CheckErr(err) is called for?